src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url(./images/star-active.svg);
+  }
```

Comment:
The current selectors will not work as intended. The `:nth-child(-n + x)` pseudo-class should be applied to the `.stars__star` class, not the `.stars--x` class. You need to change the selectors to `.stars--x .stars__star:nth-child(-n + x)`. This way, the `nth-child` pseudo-class will select the appropriate number of star elements within a `.stars--x` element.